### PR TITLE
Windows support for scripts

### PR DESCRIPTION
**Pull request by JTraill**
### ✨Startup scripts now supported on Windows machines 🌟

Scripts would fail when being run from a windows machine.
Solution was to utilize a bash terminal for the unix commands.
Also had to adjust the env script to remove incorrect windows line endings.

feat: allow scripts to support being run from windows environments
docs: adjust clone target in readme

### CODE REVIEW

**Review by jdtoombs — APPROVED** [review]
Thanks for doing this!

**Review by hansonchun-quartech — APPROVED** [review]
Working good! Ran the scripts on my local windows machine.

File: webapp/env.sh [hunk 1]
```diff
@@ -40,12 +42,12 @@ security_group=$(grep -E '^SECURITY_GROUP_ID=' .env | cut -d '=' -f2)
...
   echo "SECURITY_GROUP_ID not found in .env, using default. If you would like to override this, please add SECURITY_GROUP_ID to your .env file"
   security_group="$security_group_default"
+  # Add the security group to the env-config.js file
+  echo "  SECURITY_GROUP_ID: \"$security_group\"," >> ./env-config.js
 else
   echo "SECURITY_GROUP_ID found: $security_group"
 fi
 
-# Add the security group to the env-config.js file
-echo "  SECURITY_GROUP_ID: \"$security_group\"," >> ./env-config.js
```
Comment (mattthiessendev) [inline webapp/env.sh:48]: issue: previously this script printed security group information twice to the `env-config.js` file. How will we communicate this is an important piece of data if we only have one instance of it after init?

**Review by mattthiessendev — APPROVED** [review]
Welcome to the team ✊

**Pull request merged**